Updates for Quick start guide (#2924)
theforeman/foreman-documentation: guides/common/modules/proc_running-installer.adoc

> [id="proc_running-installer_{context}"]

Reviewer: ```suggestion
[id="running-the-{project-context}-installer_{context}"]
```

> [id="running-installer_{context}"]

Reviewer: Please ensure that the anchor, title, and file name match.

Author: Our d/s modular docs reference says not to include the prefix in the anchor ID but I do not mind including it for the sake of consistency with the other modules. Will do.

> = Running the Installer

Reviewer: ```suggestion
= Running the {Project} installer
```

> The installation run is non-interactive, but the configuration can be customized by supplying any of the options listed in `foreman-installer --help`, or by running `foreman-installer -i` for interactive mode.

Reviewer: ```suggestion
The installation run is non-interactive, but the configuration can be customized by supplying any of the options listed in `{foreman-installer} --help`, or by running `{foreman-installer} -i` for interactive mode.
```

> More examples are given in the Installation Options section.

Reviewer: I assume "Installation Options" refers to stdout from "foreman-installer --help". Does it need some emphasis then?

The `-v` option disables the progress bar and displays all changes.

.Procedure

> 

Reviewer: ```suggestion
```

* Run the following command:
+
[options="nowrap" subs="+quotes,attributes"]
----
# {installer-scenario}
----

> When the installer has completed, details are printed about where to find Foreman and the Smart Proxy.

Reviewer: ```suggestion
When the installer has completed, details are printed about where to find {ProjectServer} and {SmartProxy}.
```

I am also unsure about the wording "are printed".

Author: Agree, re: "are printed". Will check.

Author: I replaced the last sentence with the text that is used in the d/s installation guide.
